python/doc/examples/reliability_sensitivity/design_of_experiments/plot_smolyak_merge.py

```diff
@@ -113,13 +112,9 @@ def computeNumberOfSmolyakNodes(level, epsilon_a, epsilon_r):
     index += 1
 palette = ot.Drawable.BuildDefaultPalette(number_of_epsilons)
 graph.setColors(palette)
-graph.setLegendPosition("topright")
-otv.View(
-    graph,
-    figure_kw={"figsize": (4.0, 3.0)},
-    legend_kw={"bbox_to_anchor": (1.0, 1.0), "loc": "upper left"},
-)
-plt.tight_layout()
+graph.setLegendPosition("upper left")
+graph.setLegendCorner([1.0, 1.0])
+otv.View(graph)
```

Comment:
```suggestion
view = otv.View(graph, figure_kw={"figsize": (4.0, 3.0)})
```
In the spirit of @mbaudin47 's earlier comment.